RHEL6_7/storage/dasd-format/check

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+. /usr/share/preupgrade/common.sh
+#END GENERATED SECTION
+
+ARCH="$(arch)"
+
+DASDVIEW=$(which dasdview)
+DASDVIEW=${DASDVIEW:-/sbin/dasdview}
+
+GREP=$(which grep)
+GREP=${GREP:-/bin/grep}
+
+STR_IFS=$IFS
+IFS=" $(echo -n -e '\t')"
+
+function wrap_exit
+{
+	IFS=$STR_IFS
+
+	exit $1
+}
+
+[ "$ARCH" == "s390x" ] || wrap_exit $RESULT_NOT_APPLICABLE
+test -x $DASDVIEW || wrap_exit $RESULT_NOT_APPLICABLE
+test -x $GREP || wrap_exit $RESULT_NOT_APPLICABLE
+test -x $CUT || wrap_exit $RESULT_NOT_APPLICABLE
+
+
+function detect_linux_disk_layout()
+{
+	log_debug "Checking Linux Disk Layout for a device $1"
+
+	local format
+	format=$($DASDVIEW -x $1 2>/dev/null | $GREP "format :" | $CUT -f 3)
+
+	test -n "$format" || {
+		log_error "dasdview parsing failed for a device $1"
+		wrap_exit $RESULT_ERROR
+	}
+
+	log_debug "Detected Direct Access Storage Device format: $format"
+
+	[[ "$format" == *"LDL"* ]]
```

Comment:
Why the asterisks? That's unkown to me. I guess it should detect substring, but I would afraid it will try to expand path (filename) as from usual behaviour of bash. Anyway, I suggest to use rather just grep:
```suggestion
	echo "$format" | $GREP -q "[:space:]LDL[:space:]"
```